x/staking/keeper/unbonding.go

```diff
@@ -202,8 +193,7 @@ func (k Keeper) SetValidatorByUnbondingID(ctx context.Context, val types.Validat
 		return err
 	}
 
-	valKey := types.GetValidatorKey(valAddr)
-	if err = k.UnbondingIndex.Set(ctx, id, valKey); err != nil {
+	if err = k.UnbondingIndex.Set(ctx, id, valAddr); err != nil {
```

Comment:
Is this still good?

Reply:
yeah, I think so. Because valKey from `GetValidatorKey` is prefixed with ValidatorKey prefix and length of address for `valAddr`. Since we no longer need the prefixes we can directly use `valAddr`. cc @testinginprod

Reply:
I think this can be state breaking change then?

Reply:
Can we add a test using `DiffCollectionsMigration` to make sure this is not breaking?